projects/backend/s3.ts

```diff
@@ -16,7 +16,7 @@ export interface Path {
     key: string
     bucket: 'published' | 'preview'
 }
-const stage = process.env.stage || 'code'
+const stage = process.env.stage || 'prod'
```

Comment:
Is this meant to be changed?

Reply:
<img src="https://media.giphy.com/media/XsUtdIeJ0MWMo/source.gif" alt="no it isn't (picard facepalm gif)"/>

Reply:
This still needs to be changed back.

Reply:
thanks